private-templates-service/adaptivecards-templating-service/src/models/models.ts

```diff
@@ -37,6 +37,17 @@ export interface JSONResponse<T> {
   result?: T;
 }
 
+export enum SortBy {
```

Comment:
Instead of specifying _enum_ values, could it just be:

```
export enum SortBy {
  dateCreated,
  dateModified,
  name
}
```
or are the string values needed by mongo?

Reply:
yup, they're essentially keys which you search by:)